tests/PHPStan/Analyser/NodeScopeResolverTest.php

```diff
@@ -5887,6 +5887,42 @@ public function dataFunctions(): array
 				'array<int, string>|int|false',
 				'$strWordCountStrTypeIndeterminant',
 			],
+			[
+				'null',
+				'$bcdivNull',
```

Comment:
Please use the modern test approach - check NodeScopeResolverTest::testFileAsserts (in this file).